src/components/toolbar.scss

```
@@ -1,16 +1,32 @@
+@import '../common/variables.scss';
 
 .dw-toolbar {
   @apply bg-navy-blue p-0 !important;
   .v-toolbar__content{
     @apply p-0 !important;
   }
+ 
   .theme--light.v-btn:not(.v-btn--icon):not(.v-btn--flat),
   .theme--dark.v-btn:not(.v-btn--icon):not(.v-btn--flat){
     @apply border-white text-white border-solid;
     @apply bg-transparent;
     height: 34px !important;
     border-radius: 4px;
   }
+  .theme--light.v-btn.v-btn--disabled {
+    @apply text-white !important;
+    opacity: 0.4;
+    .v-icon {
+      @apply text-white !important;
+    }
+  }
+  .split-button {
+    .v-menu__activator {
+      .theme--light.v-btn.v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat):not(.v-btn--outline) {
+        background-color: $dude-red !important;
```

this can be changed to 
```suggestion
        @apply bg-dude-red !important;
```

---

also, you might be able to get away with inheriting background color here which would make it more generic to any future color schemes